.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -0,0 +1,38 @@
+name: publish apex-node-bundle
```

Comment:
@mingxuanzhangsfdx please look into abstracting this workflow into a common location